resdk/utils/resolwe.py

```diff
@@ -10,7 +10,46 @@ class ResolweUtilsMixin(object):
     def run_bamplot(self, bam, genome, input_gff=None, input_region=None, stretch_input=None,
                     color=None, sense=None, extension=None, rpm=None, yscale=None, names=None,
                     plot=None, title=None, scale=None, bed=None, multi_page=None):
-        """Run bamplot."""
+        """Run ``bamplot`` on the list of bam files.
+
+        This method runs `bamplot`_ with bams, genome and gff or region
```

Comment:
Include link to `bamplot` process. You can leave link to bamplot tool, but make clear that this function runs process on resolwe-bio which wrapps original bamplot.